Documentation/platforms/aws.md

```diff
@@ -0,0 +1,77 @@
+
+The etcd team has produced a [hardware recommendation guide](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md) which is very useful for “ballparking” how many nodes and what instance type are necessary for your cluster.
+
+AWS provides a service for creating groups of EC2 instances which are dynamically sized to match load on the instances. While running etcd cluster instances in an Auto Scaling Group ([ASG](http://docs.aws.amazon.com/autoscaling/latest/userguide/AutoScalingGroup.html)) is fine, using the ASG to dynamically change the size of the etcd cluster is not recommended. Auto Scaling Groups do provide a number of benefits besides cluster scaling which include:
+
+* distribution of EC2 instances across Availability Zones (AZs)
+* EC2 instance fail over across availability zones
+* consolidated monitoring and life cycle control of instances within an ASG
+
+The use of an ASG to create a [self healing etcd cluster](#self-healing) is one of the design considerations when deploying an etcd cluster to AWS.
+
+## Cluster Design
+
+The purpose of this section is to provide foundational guidance for deploying etcd on AWS. The discussion will be framed by the following three critical design criteria about the etcd cluster itself:
+
+* block device provider: limited to the tradeoffs between EBS or instance storage (InstanceStore)
+* cluster topology: how many nodes should make up an etcd cluster; should these nodes be distributed over multiple AZs
+* managing etcd members: creating a static cluster of EC2 instances or using an ASG.
+
+The intended cluster workload will is the primary driver of cluster design decisions. A configuration store for microservices may require different design considerations than a distributed lock service, a secrets store, or the Kubernetes control plane. Cluster design tradeoffs include considerations such as
+
+* availability
+* data safety
+* performance/throughput
+* self healing
+
+which must be matched to the expectations of the intended workload to be served by the etcd cluster.
+
+### Availability
+
+Instance availability on AWS is ultimately determined by the Amazon EC2 Region Service Level Agreement (SLA) which is the policy by which Amazon describes their precise definition of a regional outage.
+
+> “Region Unavailable” and “Region Unavailability” mean that more than one Availability Zone in which you are
+running an instance, within the same Region, is “Unavailable” to you.
+
+In the context of an etcd cluster this means you must have at minimum a three member cluster where EC2 instances are spread across at least two AZs in order for an etcd cluster to be considered highly available at a Regional level.
```

Comment:
For an etcd cluster this means there must be at minimum three members with EC2 instances spread across at least two AZs in order to be considered highly available at a Regional level.